framework/src/outputs/png/PNGOutput.C

```diff
@@ -58,34 +66,93 @@ void
 PNGOutput::makeMeshFunc()
 {
 
-  const std::vector<unsigned int> var_nums = {0};
+  if (processor_id() != 0)
+    mooseError("PNGOutput does not currently function in parallel.");
+
+  // By default _use_aux is false.
+  _use_aux = false;
```

Comment:
Move this to the ctor initialization list. Never forget to initialize a POD type - it WILL bite you!